groundstation/static/js/components/CommunicationsListFull.jsx

```diff
@@ -65,10 +113,27 @@ const CommunicationsList = (props) => {
         </div>
       )
     }
+
+    let is_admin = true;
+    const auth_token = localStorage.getItem('auth_token');
+    console.log(auth_token);
+    fetch(`/api/users/${auth_token}`).then(results => {
+        return results.json();
+    }).then(data => {
+        if (data.status === 'success') {
+            is_admin = data.data.is_admin;
+            setIsAdmin(is_admin);
+        } else {
+            console.error('Unexpected error occured:');
+        }
+    });
+    //console.log(is_admin);
```

Comment:
```suggestion
    const auth_token = localStorage.getItem('auth_token');
    console.log(auth_token);
    fetch(`/api/users/${auth_token}`).then(results => {
        return results.json();
    }).then(data => {
        if (data.status === 'success') {
            setIsAdmin(data.data.is_admin);
        } else {
            console.error('Unexpected error occured:');
        }
    });
```
Not sure if the extra `is_admin` variable is needed and it makes things kinda confusing when there is also an `isAdmin` state variable. We can maybe set the state directly and get rid of the `is_admin` variable but if this does not work out then keep it how it is.

Also, remove the commented debug print